cmd/dmverity-vhd/main.go

```diff
@@ -81,14 +81,14 @@ func fetchImageLayers(ctx *cli.Context) (layers []v1.Layer, err error) {
 	tarballPath := ctx.GlobalString(tarballFlag)
 	ref, err := name.ParseReference(image)
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to parse image reference: %s", image)
+		return nil, fmt.Errorf("failed to parse image reference %s: %w", image, err)
 	}
 
 	dockerDaemon := ctx.GlobalBool(dockerFlag)
 
 	// error check to make sure docker and tarball are not both defined
 	if dockerDaemon && tarballPath != "" {
-		return nil, errors.Errorf("cannot use both docker and tarball for image source")
+		return nil, errors.New("cannot use both docker and tarball for image source")
```

Comment:
can you also replace the `"github.com/pkg/errors"` import with `"errors"`?

Reply:
ugh, that was the intent when I made the change. but I guess I didn't fix imports.